src/server/verification/processor/provider/ZoomProvider.js

```diff
@@ -111,17 +120,24 @@ class ZoomProvider implements IEnrollmentProvider {
         // so we'll reject with isNotMatch: true instead
         // to show the error screen on app side immediately
         // notifying about liveness check failed
-        throwCustomException(message, { isNotMatch }, response)
+        throwException(message, { isNotMatch }, response)
       }
 
+      resultBlob = get(response, 'scanResultBlob')
+
       // if liveness / security issues were detected
       if ([LivenessCheckFailed, SecurityCheckFailed].includes(name)) {
         isLive = false
 
         // notifying about liveness check failed
         await notifyProcessor({ isLive })
         log.warn(message, { enrollmentIdentifier })
-        throwCustomException(message, { isLive }, response)
+        throwException(message, { isLive, resultBlob }, response)
+      }
+
+      // if had response - re-throw with scanResultBlob
+      if (response) {
+        throwException(exception, { resultBlob })
```

Comment:
what is this error case?

Reply:
@sirpy  this is any Facetec-defined error (having scanResultBlob).
In this case we have to return it and process client-side

the next one case is some unexpected error (or error don't having response/result blob)
in this case we just rethrowing it, on client side it will be .calce() with 'sorry something unexpected' screen